CHANGELOG.md

```diff
@@ -38,7 +38,9 @@ Format:
 ### Security
 - 
 -->
-<!-- ## [Unreleased] -->
+## [Unreleased]
+### Changed
+- Shorten Axios errors to make them more readable (https://github.com/SuffolkLITLab/ALKiln/pull/632)
```

Comment:
I think the issue number isn't what you intended?

```suggestion
- Shorten Axios errors to make them more readable (https://github.com/SuffolkLITLab/ALKiln/pull/605)
```

Reply:
I also liked the list of changes in the comment so I put them here, but if you don't want to include that, that's fine. Just an alternative version.

```suggestion
- Shorten Axios errors to make them more readable (https://github.com/SuffolkLITLab/ALKiln/pull/605)
- Actually fail when we log the errors
- Move `toJSON()` calls to docassemble_api_REST
- Add better comment to `process.exitCode`
```

Reply:
I do intentionally link to the PR when I fix things. Issues usually have a lot of extra discussion that don't get implemented, and the issue is still linked to from the PR, usually in the first comment. I've found it's a lot easier to find the issue fixed from a PR than finding the PR from the issue.

I normally would be fine with adding more details to the changelog, but those specific commit comments are very internal, and don't make sense outside the context of the choices made in the PR ("move toJSON calls" move them from where? from a different file they were in earlier in the PR. From the changelog comment, it looks like those toJSON calls were around before that release, but they weren't).

Reply:
Those make some sense. I'm good with whatever you choose.